recipes/dice-template-library/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class DiceTemplateLibrary(ConanFile):
+    name = "dice-template-library"
+    description = "This template library is a collection of handy template-oriented code that we, the Data Science Group at UPB, found pretty handy."
+    homepage = "https://dice-research.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("template", "template library", "compile-time", "switch", "integral tuple")
```

Comment:
```suggestion
    topics = ("template", "template-library", "compile-time", "switch", "integral-tuple")
```
search engine are a bit limited with spaces

Reply:
thx for the hint. done